app/grandchallenge/reader_studies/forms.py

```diff
@@ -412,9 +436,33 @@ def widget_choices(self):
             ) from error
         return choices
 
+    def interactive_algorithm_choices(self):
+        answer_type = self.__get_answer_type()
```

Comment:
Why is this using a double underscored method? Shouldn't this just be a property?

Reply:
Not sure why it was implemented this way. I just reused what was there to stay within the same code pattern as for the other dynamic fields on the form, but I agree that making this a property `answer_type` on the form would make sense. Should I go ahead and refactor this?

Reply:
Yes please, no need to use `__get` for a property.